src/main/java/org/cubedb/core/Cube.java

```diff
@@ -16,7 +16,7 @@
 		String fromPartition, String toPartition, List<Filter> filters, String groupBy);
 	Map<GroupedSearchResultRow, Long> get(
 		int lastRange, List<Filter> filters, String groupBy);
-	int getNumRecords();
+	//int getNumRecords();
```

Comment:
You remove the body of the method completely while still keeping the commented string here.

Please, remove the commented code.